Electrometer/v7/_base.yaml

```diff
@@ -0,0 +1,82 @@
+# Configuration for photodiode electrometer attached to comcam cart.
+# Created 2023-01-04
+
+instances:
+  - sal_index: 101
+    mode: 1
+    range: 2.0e-08
+    tcpip:
+      hostname: 'cbp-calElectrometer.cp.lsst.org'
+      port: 5024
+      timeout: 2
+    s3_instance: "cp"
+    fits_file_path: '/home/saluser/develop/electrometerFitsFiles'
+    image_name_service: "http://ccs.lsst.org"
+    filters:
+      median: false
+      general: false
+      average: false
+    sensor:
+      brand: "Hamamatsu"
+      model: "S2281"
+      serial_number: "S22818D091"
+    accessories:
+      vsource: false
+      temperature: false
+    location: "ComCam"
+    integration_time: 0.1
+    electrometer_type: "Keysight"
+    electrometer_model: "B2987A"
+    electrometer_config: {}
+  - sal_index: 102
+    mode: 1
+    range: 2.0e-08
+    tcpip:
+      hostname: 'cbp-Electrometer.cp.lsst.org'
+      port: 5024
+      timeout: 2
+    s3_instance: "cp"
+    fits_file_path: '/home/saluser/develop/electrometerFitsFiles'
+    image_name_service: "http://ccs.lsst.org"
+    filters:
+      median: false
+      general: false
+      average: false
+    sensor:
+      brand: "Hamamatsu"
+      model: "S2281"
+      serial_number: "S22818D091"
+    accessories:
+      vsource: false
+      temperature: false
+    location: "L5"
+    integration_time: 0.1
+    electrometer_type: "Keysight"
+    electrometer_model: "B2985B"
+    electrometer_config: {}
+  - sal_index: 201
+    mode: 1
+    range: 2.0e-08
+    tcpip:
+      hostname: 'auxtel-electrometer.cp.lsst.org'
+      port: 4001
+      timeout: 2
+    s3_instance: "cp"
+    fits_file_path: '/home/saluser/develop/electrometerFitsFiles'
+    image_name_service: "http://ccs.lsst.org"
+    filters:
+      median: false
+      general: false
+      average: false
+    sensor:
+      brand: "Hamamatsu"
+      model: "S2281"
+      serial_number: ""
+    accessories:
+      vsource: false
+      temperature: false
+    location: "AuxTel"
+    integration_time: 0.1
+    electrometer_type: "Keithley"
+    electrometer_model: "6517b"
+    electrometer_config: {}
```

Comment:
Need a new line at the end of file:
![image](https://github.com/user-attachments/assets/63b04501-b4dd-4e1f-9dcb-30722893192c)